src/design/plone/ioprenoto/restapi/services/bookable_list/get.py

```diff
@@ -152,7 +155,26 @@ def reply(self):
                         "title": uo.Title(),
                         "id": uo.getId(),
                         "uid": uo.UID(),
+                        "contact_info": self.get_uo_contact_info(uo),
                         "prenotazioni_folder": folders,
                     }
                 )
         return response
+
+    def get_uo_contact_info(self, uo):
+        result = []
+
+        for contact in getattr(uo, "contact_info", []):
+            if contact.isBroken():
+                logger.error(
```

Comment:
```suggestion
                logger.warning(
```

Reply:
But can we thrack the missing relation by sentry in this way?